ansible/deploy.yml

```diff
@@ -44,10 +44,43 @@
              | default([]) | subelements('physical_networks') }}
 
 - hosts: libvirt
+  vars:
+    # Allocations are stored in the localhost's vars.
+    nodes: >-
+      {{ hostvars['localhost'].allocations.result[inventory_hostname]
+           | default([]) }}
   tasks:
-    - include_tasks: libvirt_create_vms.yml
+    - name: Create Libvirt VMs
+      include_tasks: libvirt_create_vms.yml
       vars:
-        # Allocations are stored in the localhost's vars.
-        vms: >-
-          {{ hostvars['localhost'].allocations.result[inventory_hostname]
-               | default([]) }}
+        vms: "{{ nodes }}"
+
+    - name: Check that enough ports are available for Virtual BMC
```

Comment:
nit:Usually a good idea to do input validation up front, in this case before libvirt_create_vms.yml.

Reply:
Sure.